.travis.yml

```diff
@@ -16,6 +16,10 @@ language: bash
 services:
 - docker
 
+before_install:
+  - gem install package_cloud
+  - pip install --user travispy
```

Comment:
@bobvanderlinden `package_cloud` is used [below][1] in `.travis/docker_run.sh`.  I assume `travispy` is a dependency, but not sure.

[1]: https://github.com/machinekit/machinekit/pull/819/files#diff-a6816f47ee9ac2d3995237c887637d24R82